functions/common/fn_feelsAddressedByGesture.sqf

```diff
@@ -0,0 +1,28 @@
+params [
+    ["_gesturer", objNull],
+    ["_gesturingVector", []],
+    ["_unit", objNull]
+];
+
+assert(!isNull _gesturer);
+assert(!isNull _unit);
+
+if ((_gesturer distance _unit) > 100) exitWith {false};
```

Comment:
how about not hardcoding those values but stuffing the in some public var that can be changed on the fly

Reply:
should also depend on whether on foot or in vehicle